packages/core/src/renderers/dispatch.field.tsx

```diff
@@ -24,8 +24,6 @@ const Dispatch = (dispatchFieldProps: DispatchFieldProps) => {
         schema={schema}
         uischema={uischema}
         path={dispatchFieldProps.path}
-        data={dispatchFieldProps.data}
```

Comment:
why don't we need to set the data anymore?

Reply:
Because the field resolves it itself by using the path.